### PR TITLE
feat(orchestrator): display a alert dialog when the user fails to abort a running workflow

### PR DESCRIPTION
**Pull request by yzhao583**
Jira: [https://issues.redhat.com/browse/FLPATH-995](https://issues.redhat.com/browse/FLPATH-995)

Display a alert dialog when the user fails to abort a running workflow

### CODE REVIEW

File: plugins/orchestrator/src/components/WorkflowInstancePage.tsx [hunk 1]
```diff
@@ -50,6 +64,14 @@ const AbortConfirmationDialogActions = (
   </>
 );
 
+const AbortAlertDialogActions = (props: AbortAlertDialogActionsProps) => (
+  <>
+    <Button onClick={props.handleClose} color="primary">
+      Ok
```
Comment (jkilzi) [inline plugins/orchestrator/src/components/WorkflowInstancePage.tsx:70]:
```suggestion
      OK
```

**Review by jkilzi — CHANGES_REQUESTED** [review]
Just one nit :wink:

**Comment by yzhao583** [comment]
@jkilzi Thanks for the review! Committed your suggestion!

File: plugins/orchestrator/src/components/WorkflowInstancePage.tsx [hunk 2]
```diff
@@ -35,10 +35,24 @@ export type AbortConfirmationDialogActionsProps = {
   handleCancel: () => void;
 };
 
+export type AbortAlertDialogActionsProps = {
+  handleClose: () => void;
+};
+
+export type AbortAlertDialogContentProps = {
+  message?: string;
```
Comment (caponetto) [inline plugins/orchestrator/src/components/WorkflowInstancePage.tsx:43]: Can we make `message` mandatory?

Comment (yzhao583) [reply plugins/orchestrator/src/components/WorkflowInstancePage.tsx]: Done

**Review by caponetto — APPROVED** [review]

**Pull request merged**